dashboard/src/components/entitypage/EntityPageView.js

```diff
@@ -39,7 +40,7 @@ SyntaxHighlighter.registerLanguage('json', json);
 export function getFormattedSQL(sqlString = '') {
   let stringResult = sqlString;
   try {
-    stringResult = format(sqlString.replace('{{', '').replace('}}', ''), {
+    stringResult = format(sqlString.replace('{{ ', '"').replace(' }}', '"'), {
```

Comment:
Do we want this? Will it get confusing if a table actually has quotes?

Reply:
that is a possibility, but without these quotes, a table like `transactions.2024-03-27t14-53-01` will cause the sql formatter to fail and it spits out an ugly one line string.

Reply:
worst case, if a table does have quotes (like in Oracle or something) we can just check for pairs and remove one